packages/cozy-harvest-lib/src/helpers/accounts.js

```diff
@@ -94,6 +96,55 @@ export const mergeAuth = (account, authData) => ({
   auth: merge(account.auth, authData)
 })
 
+/**
+ * Gets the vault cipher relationship to an account
+ * @param  {object} account   io.cozy.accounts document
+ * @return {string}           cipher uuid
+ */
+export const getVaultCipherId = account =>
+  account.relationships &&
+  account.relationships.vaultCipher &&
+  account.relationships.vaultCipher._id
+
+/**
+ * Gets the linked cipher in vault
+ * If none is linked, search on that match
+ * @param account - io.cozy.accounts document
+ * @param {WebVaultClient} vaultClient
+ * @param {object} search criterias
+ * @param {string} search.username
+ * @param {string} search.password
+ * @param {string} search.uri
+ * @return {CipherView|null}
+ */
+export const getVaultCipher = async (
```

Comment:
I'm not sure this is the right file for this function. This is essentially a Model file for `io.cozy.accounts`, I don't think Vault helper functions should be here.

Reply:
You could see that as an helper for ciphers (it fetches ciphers) or an helper for accounts (it uses an account as first parameter and use internal data from account, public functions from ciphers). Both ways are fine with me.